src/Combinatorics/Graphs/functions.jl

```diff
@@ -588,7 +588,9 @@ julia> incidence_matrix(g)
 []
 ```
 """
-incidence_matrix(g::Graph{T}) where {T <: Union{Directed, Undirected}} = IncidenceMatrix(Polymake.graph.incidence_matrix(pm_object(g)))
+function incidence_matrix(g::Graph{T}) where {T <: Union{Directed, Undirected}}
+  IncidenceMatrix(transpose(Polymake.graph.incidence_matrix(pm_object(g))))
```

Comment:
it can now be passed to simplicial_complex correctly. I just pushed a fix for the docs. It also aligns with signed_incidence_matrix which considers the columns as edges

Reply:
this was resolved by adding a function that converts a graph to a simplicial complex, and leaving `incidence_matrix(G)` and `simplicial_complex(IM::IncidenceMatrix)` as is